packages/playground/src/weblets/tf_deployment_list.vue

```diff
@@ -187,9 +187,15 @@
           />
           <IconActionBtn
             tooltip="Open Nextcloud"
+            color="white"
             icon="mdi-web"
-            color="anchor"
-            :href="'https://' + item.value[0].env.NEXTCLOUD_DOMAIN + ':8443'"
+            :href="'https://' + item.value[0].env.NEXTCLOUD_DOMAIN"
+          />
+          <IconActionBtn
+            tooltip="Nextcloud Setup"
+            color="white"
```

Comment:
The same goes here

Reply:
Done. We set "anchor" as in all the other weblets, so it's uniform on the playground.